R/rmst.R

```diff
@@ -35,25 +50,48 @@
 #' @examples
 #' data(ex1_delayed_effect)
 #' rmst(
-#'   data = ex1_delayed_effect,
+#'   x = ex1_delayed_effect,
 #'   var_label_tte = "month",
 #'   var_label_event = "evntd",
 #'   var_label_group = "trt",
 #'   tau = 10,
 #'   reference = "0"
 #' )
-rmst <- function(
-    data,
+#'
+#' # Formula interface
+#' rmst(
+#'   month ~ evntd + trt,
```

Comment:
I have some concerns regarding whether this formula makes statistical sense. Typically, the `~` symbol is used when fitting a linear model, as in `y ~ x1 + x2`. However, in the context of `rmst()`, where the object is a survival object, I am wondering if something like a `Surv(month, event, trt)` object would be more appropriate and meaningful.

Reference: https://www.rdocumentation.org/packages/survival/versions/2.11-4/topics/Surv

Reply:
That example is to demonstrate that only the order matters. `month ~ evntd + trt` is equivalent to `Surv(month, event, trt)`. I'm open to updating the documentation in order to make this more clear to end users. I did my best to explain in the Details section that the formula syntax chosen is purely to indicate intent.

Reply:
I attempted to address your feedback in 3ccb49388c6959c38c91c0debfe1f5770968b2d4

Reply:
Thanks, John!